src/widgets/editors/markdowneditor.cpp

```diff
@@ -781,6 +790,76 @@ bool MarkdownEditor::processUrlFromMimeData(const QMimeData *p_source)
     return false;
 }
 
+bool MarkdownEditor::processMultipleMideData(const QMimeData *p_source) {
+    bool isProcess = false;
+    const auto urls = p_source->urls();
+    if (urls.size() <= 1) {
+        return isProcess;
+    }
+
+    // Judgment if all QMimeData are images.
+    bool isAllImage = false;
+    for (const QUrl &url : urls) {
+        bool isImage = PathUtils::isImageUrl(PathUtils::urlToPath(url));
+        if (isAllImage != isImage) {
+            isAllImage = isImage;
+        }
+    }
+
+    SelectDialog dialog(tr("Insert multiple files from the Clipboard"), this);
+    // It is temporarily assumed that 50 files will have a performance problem prompt.
+    if (urls.size() >= 50) {
```

Comment:
Unnecessary. And changing the window title to such long sentence is somewhat ugly.